src/main/resources/application.yml

```diff
@@ -38,6 +42,8 @@ image-manager:
   disk-log-mount-point: ${LOG_DIR}
   log-requests: ${LOG_REQUESTS}
   url-show-mode: true
+  route-to-limit: http://localhost:5000
```

Comment:
on `prod` environment we should use environmental variables, hardcoded localhost will not work, not to mention the wrong port

Reply:
connection limit should be a variable as well